app/ApiClients/BaseApi.php

```diff
@@ -30,10 +30,9 @@
 
 class BaseApi
 {
-    protected $base_uri;
-    /** @var int */
-    protected $timeout = 3;
-    private $client;
+    protected ?string $base_uri;
```

Comment:
this should not be nullable

Reply:
OK, I think I had error with it non-nullable, so I'll deal with them

Reply:
I don't understand how it can be related: https://github.com/librenms/librenms/actions/runs/3273454425/jobs/5385675803

Reply:
Because oxidized.php line 37 sets it to null